src/odemis/driver/test/xt_client_test.py

```diff
@@ -83,7 +83,7 @@
               }
 
 CONFIG_FIB_SCANNER = {"name": "fib-scanner", "role": "ion", "channel": "ion2"}
-CONFIG_FIB_SEM = {"name": "sem", "role": "sem", "address": "PYRO:Microscope@192.168.31.162:4242",
+CONFIG_FIB_SEM = {"name": "sem", "role": "sem", "address": "PYRO:Microscope@192.168.31.162:4243",
```

Comment:
That's incorrect. That "FIB_SEM" component uses xtlib to access the FIB image. So it should still be 4242... although on the testing VM, 4242 is used to run the *xttoolkit* simulator, which doesn't have a FIB component, so it's going to fail (it actually should already fail). 
In any case, better do nothing that changing to 4243, in the whole `xt_client_test` file.